src/sentry/seer/endpoints/seer_rpc.py

```diff
@@ -990,6 +992,46 @@ def send_seer_webhook(*, event_name: str, organization_id: int, payload: dict) -
     return {"success": True}
 
 
+def trigger_coding_agent_launch(
+    *,
+    organization_id: int,
+    integration_id: int,
+    run_id: int,
+    trigger_source: str = "solution",
+) -> dict:
+    """
+    Trigger a coding agent launch for an autofix run.
+
+    Args:
+        organization_id: The organization ID
+        integration_id: The coding agent integration ID
+        run_id: The autofix run ID
+        trigger_source: Either "root_cause" or "solution" (default: "solution")
+
+    Returns:
+        dict: {"success": bool, "detail": str (optional)}
+    """
+    try:
+        launch_coding_agents_for_run(
+            organization_id=organization_id,
+            integration_id=integration_id,
+            run_id=run_id,
+            trigger_source=AutofixTriggerSource(trigger_source),
+        )
+        return {"success": True}
+    except Exception as e:
+        logger.exception(
+            "coding_agent.rpc_launch_error",
+            extra={
+                "organization_id": organization_id,
+                "integration_id": integration_id,
+                "run_id": run_id,
+                "error": str(e),
+            },
+        )
+        return {"success": False, "detail": str(e)}
```

Comment:
please try and avoid adding blanket error exception handlers. additionally, please avoid adding `str(e)` to any logs -- we work at sentry so we should use sentry's conventions for capturing errors :D (either `logger.exception` which automatically includes the stacktrace, or `sentry_sdk.capture_exception`.

if we want to have something that handles exceptions across our RPC calls and returns the errors / success=false, that is okay, but we should standardize across our RPC endpoints if that is the case with a decorator or middleware.

see https://sentry.slack.com/archives/CUHS29QJ0/p1761690147925219 for more reasoning